sagemaker-debugger/tensorflow2/tensorflow2_keras_to_estimator/tf2-keras-model-to-estimator.ipynb

```diff
@@ -0,0 +1,407 @@
+{
```

Comment:
<p><a href="https://aws.amazon.com/sagemaker/" target="_blank">Amazon SageMaker</a> is a machine learning platform to build, train, and host state-of-the-art ML and AI models. <a href="https://github.com/awslabs/sagemaker-debugger" target="_blank">Amazon SageMaker Debugger</a> offers the capability to debug machine learning models during training and identifies problems with the models in real-time.</p><p> </p><p>Experimental support for TF 2.x was initially introduced in v0.7.1 of the Debugger's smdebug library. A full description of support is available at <a href="https://github.com/awslabs/sagemaker-debugger/tree/master/docs/tensorflow.md" target="_blank">Amazon SageMaker Debugger with TensorFlow</a>.</p><p> </p><p>With the smdebug v0.9.0 release, its support has been extended to cover TF 2.x <code><a href="https://www.tensorflow.org/api_docs/python/tf/keras/estimator/model_to_estimator" target="_blank">model_to_estimator</a></code> and Estimator APIs.</p><p> </p><p>In this notebook, you will learn how to run your training job with a python script with the TF 2.x <code><a href="https://www.tensorflow.org/api_docs/python/tf/keras/estimator/model_to_estimator" target="_blank">model_to_estimator</a></code> API, debug the training job using the Debugger built-in rules,</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/1298/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>